cesm/flux_atmocn/shr_flux_mod.F90

```diff
@@ -0,0 +1,2299 @@
+module shr_flux_mod
```

Comment:
Why was this file put into a new directory?

Reply:
Because CAM needs it for physconst mod. And it is its own directory since it relates to atm/ocn fluxes. I thought it would be a good idea to separate that out explicitly.